app/assets/javascripts/controllers/ops/diagnostics_database_form_controller.js

```diff
@@ -98,6 +108,23 @@ ManageIQ.angular.app.controller('diagnosticsDatabaseFormController', ['$http', '
       (vm.diagnosticsDatabaseModel.log_aws_region === '' || typeof vm.diagnosticsDatabaseModel.log_aws_region === 'undefined'));
   };
 
+  vm.securityProtocolSelect = function() {
+    return vm.diagnosticsDatabaseModel.log_protocol === "OpenStack Swift";
+  };
+
+  vm.swiftSecurityProtocolSelect = function() {
+    return vm.diagnosticsDatabaseModel.log_protocol === 'OpenStack Swift';
```

Comment:
Use `return vm.scheduleModel.action_typ == "db_backup" && vm.scheduleModel.log_protocol === 'OpenStack Swift';` to prevent error in [comment](https://github.com/ManageIQ/manageiq-ui-classic/pull/4684/files#r222678296)

Reply:
I assume you mean diagnosticsDatabaseModel?

Reply:
Yes. Got them mixed up.